docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -0,0 +1,281 @@
+
+    <sect2 id="CustTutorial1" role="nobrk">
+      <title>Create a Custom Configuration Chart for Two Roxies</title>
+
+      <orderedlist>
+        <listitem>
+          <para>Create a new text file and name it <emphasis
+          role="bold">TwoRoxies.yaml</emphasis> and open it in a text
+          editor.</para>
+
+          <para>You can use any text editor.</para>
+        </listitem>
+
+        <listitem>
+          <para>Prepare to copy portions of default YAML file
+          (..\helm\hpcc\values.yaml) </para>
+
+          <para>You can use the HPCC-Platform repository or the helm-chart
+          repository. </para>
+
+          <para>https://github.com/hpcc-systems/HPCC-Platform </para>
+
+          <para>https://github.com/hpcc-systems/helm-chart </para>
+        </listitem>
+
+        <listitem>
+          <para>Copy the entire <emphasis role="bold">roxie:</emphasis>
+          section and paste it into the new TwoRoxies.yaml file.</para>
+        </listitem>
+
+        <listitem>
+          <para>Copy the entire contents, except the first line, and paste it
+          at the end of the file.</para>
+        </listitem>
+
+        <listitem>
+          <para>In the second block, edit the value for <emphasis
+          role="bold">name:</emphasis> and change it to <emphasis
+          role="bold">roxie2</emphasis></para>
+        </listitem>
+
+        <listitem>
+          <para>In the second block, edit the value for <emphasis
+          role="bold">name:</emphasis> under <emphasis
+          role="bold">services:</emphasis> and change it to <emphasis
+          role="bold">roxie2</emphasis>.</para>
+        </listitem>
+
+        <listitem>
+          <para>Save the file and close the text editor.</para>
+
+          <para>The resulting TwoRoxies.yaml file should look like this</para>
+
+          <para><emphasis role="bold">Note:</emphasis> The comments have been
+          removed to simplify the example:</para>
+
+          <para><programlisting>roxie:
+- name: roxie
+  disabled: false
+  prefix: roxie
+  services:
+  - name: roxie
+    servicePort: 9876
+    listenQueue: 200
+    numThreads: 30
+    visibility: local
+  replicas: 2  
+  numChannels: 2
+  serverReplicas: 0
+  localAgent: false
+  traceLevel: 1
+  topoServer:
+    replicas: 1
+
+- name: roxie2
+  disabled: false
+  prefix: roxie2
+  services:
+  - name: roxie2
+    servicePort: 9876
+    listenQueue: 200
+    numThreads: 30
+    visibility: local
+  replicas: 2  
+  numChannels: 2
+  serverReplicas: 0
+  localAgent: false
+  traceLevel: 1
+  topoServer:
+    replicas: 1
+</programlisting></para>
+        </listitem>
+      </orderedlist>
+
+      <para><emphasis role="bold">Deploy using the new custom configuration
+      chart.</emphasis></para>
+
+      <orderedlist>
+        <listitem>
+          <para>Open a terminal and navigate to the <emphasis
+          role="bold">helm</emphasis> folder of the repository you
+          downloaded.</para>
```

Comment:
as noted above/previously. 99% of the time, users should not be using a copy of the repository sources, but should instead be using the charts via the local repository.
e.g. general usage should be :


1) helm repo add hpcc https://hpcc-systems.github.io/helm-chart/

2) (within any directory of users choice)
helm show values hpcc/hpcc > hpccdefaults.yaml

edit or use as basis to create supplement custom yaml files.

3) pass in to helm install, pointing to the repo chart (not sources) e.g.:

helm install myhpcc hpcc/hpcc -f tworoxies.yaml

NB: hpcc/hpcc = <repository-name>/<chart-name>